helm/zot/values.schema.json

```diff
@@ -1,412 +1,864 @@
 {
-    "$schema": "http://json-schema.org/schema#",
-    "type": "object",
-    "properties": {
-        "configFiles": {
-            "type": "object",
-            "properties": {
-                "config.json": {
-                    "type": "string"
+  "additionalProperties": false,
```

Comment:
The fact that `"additionalProperties": false` is set on every object is a significant change compared to our current practice. I think it makes sense, as it's the only way to catch properties (for example that are misspelled) which are not defined in the schema.

it also means that if we inject configuration values from other layers, like from a controller, or from the catalog, or maybe even from subcharts / library charts, these must be defined in the schema.

Reply:
https://github.com/dadav/helm-schema?tab=readme-ov-file#additionalproperties

I think we can make the transition smoother